rust/core-lib/src/tabs.rs

```diff
@@ -697,12 +697,14 @@ impl CoreState {
 
     /// Handles a file system event related to a currently open file
     #[cfg(feature = "notify")]
-    fn handle_open_file_fs_event(&mut self, event: DebouncedEvent) {
-        use notify::DebouncedEvent::*;
-        let path = match event {
-            NoticeWrite(ref path) | Create(ref path) | Write(ref path) | Chmod(ref path) => path,
+    fn handle_open_file_fs_event(&mut self, event: Event) {
+        use notify::event::*;
+        let path = match event.kind {
+            EventKind::Create(CreateKind::Any)
```

Comment:
If what I just asserted is true e.g. this wouldn't work since we want to match `CreateKind::File` and `CreateKind::Folder` here